docs/providers/aws/guide/serverless.yml.md

```diff
@@ -142,6 +142,7 @@ provider:
       level: INFO # Optional configuration which specifies the log level to use for execution logging. May be set to either INFO or ERROR.
       fullExecutionData: true # Optional configuration which specifies whether or not to log full requests/responses for execution logging. Defaults to true.
     websocket: true # Optional configuration which specifies if Websockets logs are used
+    frameworkLambda: true # Optional, whether to write CloudWatch logs for custom resource lambdas as added by the framework
```

Comment:
Maybe rename this to `customResourceLambda` (or something along those lines) to make this more expressive?

Reply:
I wasn't sure about `customResourceLambda` as user may configure it's own custom resources, and then this name might turn confusing.

We can do it `frameworkCustomResoureLambda`, still I think I settled on `frameworkLambda` to cover generally all lambdas which might be generated by framework (Still not sure whether we may have any other use case apart of custom resources).

What do you think?

Reply:
Ok, let's go with `frameworkLambda` :+1: